openfl-tutorials/interactive_api/Tensorflow_CIFAR_tfdata/envoy/envoy_config_two.yaml

```diff
@@ -0,0 +1,9 @@
+params:
```

Comment:
We can get rid of this file since README describes copying this from the first envoy.

Reply:
3c94773